docs/src/main/sphinx/release/release-477.md

```diff
@@ -0,0 +1,291 @@
+  diagnostics information regarding adaptive planner operation. ({issue}`26274`)
+* Add the `coordinatorId` to the `/v1/info` endpoint. ({issue}`23910`)
+* Add support for [`ALTER VIEW ... REFRESH`](/sql/alter-view).
+  ({issue}`25906`)
+* Add support for branching.
+  ({issue}`25751`, {issue}`26300`, {issue}`26136`)
+* Add physical data scan tracking to resource groups. ({issue}`25003`)
+* Add DNS node inventory for use in K8s-like environments that provide a DNS
+  name for all workers. ({issue}`26119`)
+* Add `discovery.type` to set node inventory system. The value can be
+  `announce`, `dns` or `airlift-discovery`. ({issue}`26119`)
+* Add the {func}`cosine_distance` function for sparse vectors. ({issue}`24027`)
+* Improve reliability of spilling for aggregation. ({issue}`25892`)
+* Improve spilling reliability for join queries. ({issue}`25976`)
+* Improve precision and scale inference for arithmetic operations between
+  decimal values. ({issue}`26422`)
+* Improve stability for join queries. ({issue}`26142`)
+* Fix correctness issues when unspilling hash aggregation by using correct hash
+  values, avoiding aggregation miscalculations. ({issue}`25892`)
+* Reduce memory consumed by grouped `ORDER BY ... LIMIT` and row number
+  operations. ({issue}`25946`)
+* Reduce memory usage of aggregations. ({issue}`25879`)
+* Fix failure when aggregation exists in other expressions in `GROUP BY AUTO`.
+  ({issue}`25987`)
+* Fix incorrect results with spatial joins. ({issue}`26021`)
+* Partition large pages to avoid `OutOfMemoryError` while serializing data in
+  the spooling protocol. ({issue}`25999`)
+* Fix spill files leaking in aggregation queries. ({issue}`26141`)
+* Reduce memory required for distinct and ordered grouped aggregations.
+  ({issue}`26276`)
+* Fix memory tracking for ordered grouped aggregations. ({issue}`26276`)
+* Fix access control check when access is granted through groups in
+  `SET SESSION AUTHORIZATION`. ({issue}`26344`)
+
+## Security
+
+## Web UI
+
+* Enhance UI responsiveness for Trino clusters without external network
+  access. ({issue}`26031`)
+* Add query details page to the Preview Web UI. ({issue}`25554`)
+
+## JDBC driver
+
+* Send detailed client information such as user-agent in the source.
+  ({issue}`25889`)
+
+## Docker image
+
+## CLI
+
+* Send detailed client information such as user-agent in the source.
+ ({issue}`25889`)
+* Allow configuring `--max-buffered-rows` and `--max-queued-rows`.
+  ({issue}`26015`)
+* Use actual stage id instead incremental number in progress output.
+  ({issue}`26139`)
+* Add support for keyboard navigation with {kbd}`Alt+↑` or {kbd}`Alt+↓` in query
+  history. ({issue}`26138`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Add support for using GCS without credentials. ({issue}`25810`)
+* Add ability to detect resource leakage in the runtime. ({issue}`26087`)
```

Comment:
What kind of resource leakage? Which runtime? What happens when leakage is detected? What's the user-visible behavior?

Reply:
@wendigo